lms/static/styles/buttons.scss

```diff
@@ -0,0 +1,40 @@
+@use "sass:map";
+
+// Button styling for LMS extending common button-component styles
+@use '@hypothesis/frontend-shared/styles/components/buttons';
+
+@use 'variables' as var;
+
+$input-button-colors: map.merge(
+  buttons.colors-for('IconButton'),
+  (
+    'background': var.$grey-2,
+    'hover-background': var.$grey-3,
+    'disabled-foreground': var.$grey-4,
+  )
+);
+
+// An icon-only button that sits next to a text input field.
+@mixin InputButtonBase {
+  $-options: (
+    'colormap': $input-button-colors,
+    'withVariants': false,
+  );
+  @include buttons.IconButton($-options) {
+    border: 1px solid var.$grey-3;
+    border-radius: 0; // Turn off border-radius to align with <input> edges
+    height: 100%;
```

Comment:
Because we don;'t have `.u-layout-row` in LMS yet, this is a bit of a hack to allow the button to fill up the parent container and match the height of the adject input. 

Better, would be to use `u-layout-row`, but I was holding off on that for now